crates/apps/src/lib/client/rpc.rs

```diff
@@ -1538,15 +1538,33 @@ pub async fn query_protocol_parameters(
     let pos_params = query_pos_parameters(context.client()).await;
```

Comment:
a small improvement - we could use `let PosParams { } = query ...` syntax to destruct the fields to make sure we're using all so if new one is added we don't miss it here